environments.json

```diff
@@ -1269,7 +1269,7 @@
     "short": "FF 60 ESR",
     "release": "2018-05-09",
     "retire": "2019-10-22",
-    "obsolete": "very"
+    "obsolete": true
```

Comment:
why?

Reply:
The "rule" I was using was "1 year after retire date".  This will be true until the next release.

Reply:
I could be wrong, but the rule was about release dates. Anyway, I don't think we should add 60 ESR to the table

Reply:
FF60 removed.